includes/wikia/models/DesignSystem/WdsExternalImage.class.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class WdsExternalImage {
```

Comment:
how can I make field named `image-data` in a response with this approach? it is not allowed name for field